bugwarrior/services/jira.py

```diff
@@ -91,6 +97,11 @@ def get_summary(self):
             return self.record['fields']['summary']['value']
         return self.record['fields']['summary']
 
+    def get_estimate(self):
+        if self.extra.get('jira_version') == 4:
+            return self.record['fields']['timeestimate']['value']
+        return self.record['fields']['timeestimate'] / 60 / 60
```

Comment:
Are we guaranteed to have a numeric value here? Maybe we should catch `TypeError` and `KeyError' so we can return `None` if either the value is null or that key doesn't exist.